core/idl/wallet/ethereum/erc20.djinni

```diff
@@ -1,59 +1,72 @@
 @import "../../math/big_int.djinni"
 @import "../../ethereum/addresses.djinni"
-@import "../amount.djinni"
 
 # Representation of ERC20Tokens.
 ERC20Token = record {
     # Name of the currency.
-	name: string;
+    name: string;
     # Symbol representation of the currency.
-	symbol: string;
+    symbol: string;
     # Address of the smart contract for this ERC20.
-	contractAddress: string;
+    contractAddress: string;
     # Maximum number of decimals (used to show the human representation of the currency).
-	numberOfDecimal: i32;
+    numberOfDecimal: i32;
+}
+
+# ERC20-like balance history.
+ERC20LikeBalanceHistory = record {
+    # Period used when representing the history.
+    period: TimePeriod;
+    # Start date of the history.
```

Comment:
Can you please include into the comments are startDate and endDate included?

Reply:
Sure.